.github/workflows/update-core-deps.yml

```diff
@@ -12,6 +12,13 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
+        with:
+          submodules: true  # Ensure submodules are checked out
+
+      - name: Update submodules
+        working-directory: inputfiles/mdn  # Replace with actual submodule path
```

Comment:
these comments look left over from an example, since `inputfiles/mdn` is the actual submodule path. Can you remove them?